richie/settings.py

```diff
@@ -234,6 +272,49 @@ class Base(DRFMixin, ElasticSearchMixin, Configuration):
         "easy_thumbnails.processors.background",
     )
 
+    # Sentry
+    #
+    # Nota bene: the DJANGO_SENTRY_DSN environment variable should be defined to
+    # report failures to Sentry
+    RAVEN_CONFIG = {
+        "dsn": values.Value(environ_name="SENTRY_DSN"),
+        "release": get_release(),
+        "environment": values.Value(environ_name="CONFIGURATION"),
+    }
+
+    LOGGING = {
+        "version": 1,
+        "disable_existing_loggers": True,
+        "root": {"level": "WARNING", "handlers": ["sentry"]},
+        "formatters": {
+            "verbose": {
+                "format": "%(levelname)s %(asctime)s %(module)s "
+                "%(process)d %(thread)d %(message)s"
+            }
+        },
+        "handlers": {
+            "sentry": {
+                "level": "ERROR",  # To capture more than ERROR, change to WARNING, INFO, etc.
+                "class": "raven.contrib.django.raven_compat.handlers.SentryHandler",
+                "tags": {"custom-tag": "x"},
+            },
+            "console": {
+                "level": "DEBUG",
+                "class": "logging.StreamHandler",
+                "formatter": "verbose",
+            },
+        },
+        "loggers": {
+            "django.db.backends": {
+                "level": "ERROR", "handlers": ["console"], "propagate": False
+            },
+            "raven": {"level": "DEBUG", "handlers": ["console"], "propagate": False},
+            "sentry.errors": {
+                "level": "DEBUG", "handlers": ["console"], "propagate": False
```

Comment:
If we set `propagate: False` here, does that not mean that the `"sentry"` handler will not receive events from this logger?

Reply:
The `sentry` handler is the root logger, hence it will receive all logs who's level is `> WARNING`. This particular logger is using the `console` handler at a `DEBUG` level with propagate set to `False`, consequently the `sentry` handler will not receive logs `>= DEBUG, <WARNING` for this particular logger.

To make a long explanation short: yes.

:wink:

Reply:
Ok then. I'm not familiar with this and I was trying to understand things by reading the code and referring to the docs.

Thank you

Reply:
That was a pretty interesting remark. Thank you for asking.